>>> 'Pkg.add("GeographicLibPy")' log
INFO: Cloning cache of GeographicLibPy from https://github.com/kshramt/GeographicLibPy.jl.git
INFO: Installing BinDeps v0.4.2
INFO: Installing Conda v0.2.3
INFO: Installing GeographicLibPy v0.0.3
INFO: Installing MacroTools v0.3.0
INFO: Installing PyCall v1.7.0
INFO: Installing SHA v0.2.0
INFO: Installing URIParser v0.1.6
INFO: Building PyCall
INFO: Recompiling stale cache file /home/vagrant/.julia/lib/v0.5/Conda.ji for module Conda.
WARNING: `@windows` is deprecated, use `@static is_windows()` instead
 in depwarn(::String, ::Symbol) at ./deprecated.jl:64
 in @windows(::Any, ::Any) at ./deprecated.jl:473
 in include_from_node1(::String) at ./loading.jl:426
 in evalfile(::String, ::Array{String,1}) at ./loading.jl:442 (repeats 2 times)
 in cd(::##2#4, ::String) at ./file.jl:59
 in (::##1#3)(::IOStream) at ./none:13
 in open(::##1#3, ::String, ::String) at ./iostream.jl:113
 in eval(::Module, ::Any) at ./boot.jl:234
 in process_options(::Base.JLOptions) at ./client.jl:239
 in _start() at ./client.jl:318
while loading /home/vagrant/.julia/v0.5/PyCall/deps/build.jl, in expression starting on line 17
INFO: No system-wide Python was found; got the following error:
could not spawn `'' -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('VERSION'))"`: no such file or directory (ENOENT)
using the Python distribution in the Conda package
Fetching package metadata .........
Solving package specifications: ..........

# All requested packages already installed.
# packages in environment at /home/vagrant/.julia/v0.5/Conda/deps/usr:
#
numpy                     1.11.1                   py27_0  
INFO: PyCall is using /home/vagrant/.julia/v0.5/Conda/deps/usr/bin/python (Python 2.7.12) at /home/vagrant/.julia/v0.5/Conda/deps/usr/bin/python, libpython = /home/vagrant/.julia/v0.5/Conda/deps/usr/lib/libpython2.7.so
INFO: Package database updated
INFO: METADATA is out-of-date â€” you may not have the latest version of GeographicLibPy
INFO: Use `Pkg.update()` to get the latest versions of your packages

>>> 'Pkg.test("GeographicLibPy")' log
Julia Version 0.5.0-rc1+1
Commit acfd04c (2016-08-05 15:23 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  WORD_SIZE: 64
           Ubuntu 14.04.5 LTS
  uname: Linux 3.13.0-92-generic #139-Ubuntu SMP Tue Jun 28 20:42:26 UTC 2016 x86_64 x86_64
Memory: 2.4471054077148438 GB (1663.50390625 MB free)
Uptime: 21055.0 sec
Load Avg:  1.33056640625  1.2392578125  1.13671875
Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz: 
       speed         user         nice          sys         idle          irq
#1  3500 MHz    1092392 s       5892 s     117001 s     603620 s         42 s
#2  3500 MHz     409731 s        774 s      59664 s    1543776 s          0 s

  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1 (ORCJIT, haswell)
Environment:
  TERM = vt100
  LD_LIBRARY_PATH = :/usr/local/lib/
  PATH = /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/vagrant/julia/bin/
  JAVA_HOME = /usr/lib/jvm/java-7-oracle
  HOME = /home/vagrant

Package Directory: /home/vagrant/.julia/v0.5
2 required packages:
 - GeographicLibPy               0.0.3
 - JSON                          0.6.0
7 additional packages:
 - BinDeps                       0.4.2
 - Compat                        0.8.6
 - Conda                         0.2.3
 - MacroTools                    0.3.0
 - PyCall                        1.7.0
 - SHA                           0.2.0
 - URIParser                     0.1.6
INFO: Testing GeographicLibPy
INFO: Recompiling stale cache file /home/vagrant/.julia/lib/v0.5/PyCall.ji for module PyCall.

WARNING: deprecated syntax "global gui::Symbol".
Use "typeassert" instead.

WARNING: deprecated syntax "global gui::Symbol".
Use "typeassert" instead.

WARNING: deprecated syntax "global npy_initialized::Bool".
Use "typeassert" instead.
WARNING: symbol is deprecated, use Symbol instead.
 in depwarn(::String, ::Symbol) at ./deprecated.jl:64
 in symbol(::SubString{String}, ::Vararg{SubString{String},N}) at ./deprecated.jl:30
 in bname(::Symbol) at /home/vagrant/.julia/v0.5/MacroTools/src/match.jl:17
 in allbindings(::Symbol, ::Array{Any,1}) at /home/vagrant/.julia/v0.5/MacroTools/src/macro.jl:1
 in collect_to!(::Array{Array{Any,1},1}, ::Base.Generator{Array{Symbol,1},MacroTools.##1#2{Array{Any,1}}}, ::Int64, ::Int64) at ./array.jl:307
 in collect_to_with_first!(::Array{Array{Any,1},1}, ::Array{Any,1}, ::Base.Generator{Array{Symbol,1},MacroTools.##1#2{Array{Any,1}}}, ::Int64) at ./array.jl:294
 in _collect(::Array{Symbol,1}, ::Base.Generator{Array{Symbol,1},MacroTools.##1#2{Array{Any,1}}}, ::Base.EltypeUnknown, ::Base.HasShape) at ./array.jl:288
 in map(::Function, ::Array{Symbol,1}) at ./abstractarray.jl:1429
 in allbindings(::Expr, ::Array{Any,1}) at /home/vagrant/.julia/v0.5/MacroTools/src/macro.jl:1
 in map(::MacroTools.##1#2{Array{Any,1}}, ::Array{Any,1}) at ./essentials.jl:124
 in allbindings(::Expr, ::Array{Any,1}) at /home/vagrant/.julia/v0.5/MacroTools/src/macro.jl:1
 in map(::MacroTools.##1#2{Array{Any,1}}, ::Array{Any,1}) at ./essentials.jl:124
 in allbindings(::Expr, ::Array{Any,1}) at /home/vagrant/.julia/v0.5/MacroTools/src/macro.jl:1
 in allbindings(::Expr) at /home/vagrant/.julia/v0.5/MacroTools/src/macro.jl:10
 in @capture(::Any, ::Any) at /home/vagrant/.julia/v0.5/MacroTools/src/macro.jl:53
 in include_from_node1(::String) at ./loading.jl:426 (repeats 2 times)
 in macro expansion; at ./none:2 [inlined]
 in anonymous at ./<missing>:?
 in eval(::Module, ::Any) at ./boot.jl:234
 in process_options(::Base.JLOptions) at ./client.jl:239
 in _start() at ./client.jl:318
while loading /home/vagrant/.julia/v0.5/PyCall/src/pyclass.jl, in expression starting on line 45

WARNING: deprecated syntax "global pyio_initialized::Bool".
Use "typeassert" instead.
ERROR: LoadError: InitError: PyError (:PyImport_ImportModule) <type 'exceptions.ImportError'>
ImportError('No module named geographiclib',)

 in pyerr_check at /home/vagrant/.julia/v0.5/PyCall/src/exception.jl:56 [inlined]
 in pyerr_check at /home/vagrant/.julia/v0.5/PyCall/src/exception.jl:61 [inlined]
 in macro expansion at /home/vagrant/.julia/v0.5/PyCall/src/exception.jl:81 [inlined]
 in pyimport(::String) at /home/vagrant/.julia/v0.5/PyCall/src/PyCall.jl:386
 in __init__() at /home/vagrant/.julia/v0.5/GeographicLibPy/src/GeographicLibPy.jl:16
 in _require_from_serialized(::Int64, ::Symbol, ::String, ::Bool) at ./loading.jl:174
 in require(::Symbol) at ./loading.jl:365
 in include_from_node1(::String) at ./loading.jl:426
 in process_options(::Base.JLOptions) at ./client.jl:262
 in _start() at ./client.jl:318
during initialization of module GeographicLibPy
while loading /home/vagrant/.julia/v0.5/GeographicLibPy/test/runtests.jl, in expression starting on line 3
===========================[ ERROR: GeographicLibPy ]===========================

failed process: Process(`/home/vagrant/julia/bin/julia -Cx86-64 -J/home/vagrant/julia/lib/julia/sys.so --compile=yes --depwarn=yes --check-bounds=yes --code-coverage=none --color=no --compilecache=yes /home/vagrant/.julia/v0.5/GeographicLibPy/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: GeographicLibPy had test errors
 in #test#61(::Bool, ::Function, ::Array{AbstractString,1}) at ./pkg/entry.jl:740
 in (::Base.Pkg.Entry.#kw##test)(::Array{Any,1}, ::Base.Pkg.Entry.#test, ::Array{AbstractString,1}) at ./<missing>:0
 in (::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}})() at ./pkg/dir.jl:31
 in cd(::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}}, ::String) at ./file.jl:59
 in #cd#1(::Array{Any,1}, ::Function, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./pkg/dir.jl:31
 in (::Base.Pkg.Dir.#kw##cd)(::Array{Any,1}, ::Base.Pkg.Dir.#cd, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./<missing>:0
 in #test#3(::Bool, ::Function, ::String, ::Vararg{String,N}) at ./pkg/pkg.jl:258
 in test(::String, ::Vararg{String,N}) at ./pkg/pkg.jl:258
 in eval(::Module, ::Any) at ./boot.jl:234
 in process_options(::Base.JLOptions) at ./client.jl:239
 in _start() at ./client.jl:318

>>> End of log
